arm-datalake-store/filesystem/2016-11-01/swagger/filesystem.json

```diff
@@ -131,15 +87,29 @@
             ]
           },
           {
-            "$ref": "#/parameters/ApiVersionParameter"
+            "name": "Transfer-Encoding",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Indicates the data being sent to the server is being streamed in chunks.",
+            "enum": [
+              "chunked"
+            ]
```

Comment:
No because this is a constant, so having a single enum results in AutoRest turning the value into a constant string. Unless that changed and we need to use x-ms-enum now, but we have been just using "enum" for the past several releases for these things.